src/lib/bootloader/language.rb

```diff
@@ -0,0 +1,19 @@
+require "cfa/base_model"
+require "cfa/augeas_parser"
+
+module Bootloader
+  # read only model to get system language from sysconfig
+  # Uses CFA base model with augeas parser
+  class Language < CFA::BaseModel
```

Comment:
this means, that bootloader package will contain sysconfig directory in lib. I am not sure if I want to have it in bootloader. BTW I will be happy if such functionality already exist in YaST in easy way how to get it, but I do not find it.